ob1k-cache/src/main/java/com/outbrain/ob1k/cache/LocalAsyncCache.java

```diff
@@ -215,6 +219,17 @@ private void unloadErrorsFromCache(final ImmutableMap<K, ComposableFuture<V>> in
   }
 
   @Override
+  public ComposableFuture<Boolean> putIfAbsentAsync(final K key, final V value) {
+    if (loadingCache != null) {
```

Comment:
I don't want to invest any effort in this class, just using the same code style as used in other methods. As we discussed, this class should anyway be refactored.